devdocs/troubleshooting-guide.md

```diff
@@ -0,0 +1,187 @@
+# Developer Troubleshooting Guide
+
+1. [Troubleshooting project setup](#troubleshooting-project-setup)
+    * [Setup checklist](#settup-checklist)
+    * [Common errors and solutions](#common-errors-and-solutions)
+2. [Troubleshooting test failures](#troubleshooting-test-failures)
+    * [Optimizing IDE layout for testing](#optimizing-ide-layout-for-testing)
+    * [How automated Browser tests work](#how-automated-Browser-tests-work)
+    * [Getting help to resolve test failures](#getting-help-to-resolve-test-failures)
+    * [Common errors and solutions](#common-errors-and-solutions)
+        * [Errors specific to Linux](#errors-specific-to-linux)
+
+This document can help you to fix the problems encountered while contributing to TEAMMATES.Failing that, you can post in the teammates-contributors to ask for help.Remember to supply as much relevant information as possible when asking for help. e.g. Which operating system are you using.
```

Comment:
We don't use teammates-contributors anymore, change it to:
...you can post in the ~~teammates-contributors~~ issue tracker to ask...
Also wrap the words "post in the issue tracker" with [this link](https://github.com/TEAMMATES/teammates/issues/new).